tokens/spacing.json

```diff
@@ -20,48 +20,68 @@
       "comment": "Grid column size"
     },
     {
-      "name": "SPACING_XX_SMALL",
-      "value": "0.444rem",
+      "name": "SPACING_1",
+      "value": "0.25rem",
       "comment": "For micro adjustments"
     },
     {
-      "name": "SPACING_X_SMALL",
-      "value": "0.667rem",
+      "name": "SPACING_2",
+      "value": "0.5rem",
       "comment": "For micro adjustments"
     },
     {
-      "name": "SPACING_SMALL",
+      "name": "SPACING_3",
+      "value": "0.75rem",
+      "comment": "For paddings in labels / inline stuff"
+    },
+    {
+      "name": "SPACING_4",
       "value": "1rem",
       "comment": "For paddings in labels / inline stuff"
     },
     {
-      "name": "SPACING_NORMAL",
+      "name": "SPACING_5",
+      "value": "1.25rem",
+      "comment": "For paddings in labels / inline stuff"
+    },
+    {
+      "name": "SPACING_6",
       "value": "1.5rem",
       "comment": "Regular padding for compact tables and such."
```

Comment:
We need to remove all comments for now, they make no sense anymore since we changed the values around, and there's duplication.

Reply:
Yeah I know, I didn't bother changing them since we first had to decide on the scale. But good catch!